recipe/meta.yaml

```diff
@@ -32,7 +32,6 @@ about:
   summary: 'fastText - Library for efficient text classification and representation learning'
   dev_url: https://github.com/facebookresearch/fastText
   doc_url: https://fasttext.cc/docs/en/support.html
```

Comment:
don't you need `license: MIT` or whatever here?

Reply:
It's already there, in line 29